src/spikeinterface/preprocessing/filter.py

```diff
@@ -10,15 +10,14 @@
 
 _common_filter_docs = """**filter_kwargs : dict
         Certain keyword arguments for `scipy.signal` filters:
-        filter_order : order
-            The order of the filter
-        filter_mode :  "sos" | "ba", default: "sos"
-            Filter form of the filter coefficients:
-            - second-order sections ("sos")
-            - numerator/denominator : ("ba")
-        ftype : str, default: "butter"
-            Filter type for `scipy.signal.iirfilter` e.g. "butter", "cheby1".
-    """
+            filter_order : order
+                The order of the filter
```

Comment:
```suggestion
                The order of the filter. Note as filtering is applied with scipy's `filtfilt` functions (i.e. acausal, zero-phase) the effective order will be double the `filter_order`.
```

Reply:
This however might be more relevant for #2942, at present all filters are applied with `filtfilt` but that will introduce causal `filt` and the docstring should be expanded to highlight the difference in cases.

Reply:
Just to check. You're saying we should implement the change you suggest here, but we will need to add more details once #2942 is merged?

Reply:
I left it ambiguous as I couldn't decide 😅 . If your happy with the suggestion I'd say let's go for it here and we can update on #2942 when required.